Exercise_A/index.html

```diff
@@ -6,33 +6,217 @@
 
     <title>Solution for day 4 using Bootstrap v4</title>
 
-    <!-- 
-      This is where we declare all of the external libraries our page uses
-      
-      You will need:
-      Animate CSS
-      Bootstrap CSS
-      You own custom stylesheet
-      FontAwesome
-
-      Refer to the Practice directory
-    -->
+    <link rel="stylesheet" href="assets/css/animate.css">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" 
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <script src="https://kit.fontawesome.com/35e4601671.js" crossorigin="anonymous"></script>
+
   </head>
 
   <!-- Nav bar setup -->
 
   <body data-spy="scroll" data-target=".navbar">
     <!-- data-spy="scroll" data-target=".navbar" data-offset="50" -->
     <!--Allows us to track the progress of the scroll -->
-    <!-- Nav bar items -->
 
-    <!-- Can use Jumbotron -->
-    <!-- Second Section -->
+    <nav class="navbar navbar-expand-lg navbar-light justify-content-between
+    fixed-top">
+      <a class="navbar-brand" href="#">
+          <p>THE ONLY BOOTSTRAP PAGE</p>

```

Comment:
Fix you navigation bar button positions as well as the style of the link elements and brand element